src/amp-story-player/README.md

```diff
@@ -84,15 +84,15 @@ There are many ways in which you can use the player on your own surfaces. Below
 <tbody>
   <tr>
     <td><strong>Carousel with circle avatars</strong> <br>In this codepen, we provide an example of how to create a carousel entry point and how to link it to your player. <br> <a href="https://codepen.io/maenrique/pen/wvWjNYr" target="_blank" rel="noopener noreferrer">https://codepen.io/maenrique/pen/wvWjNYr</a><br><br><strong>Demo</strong>  <br>This demo showcases a more complex implementation with a horizontally scrollable carousel and the player being opened in a lightbox. <br><a href="https://web-player-cards-carousel.web.app/examples/amp-story/player-entry-point-circular.html" target="_blank" rel="noopener noreferrer">https://web-player-cards-carousel.web.app/examples/amp-story/player-entry-point-circular.html</a><br></td>
-    <td><img width="360" height="642" src="https://github.com/ampproject/amphtml/blob/master/src/amp-story-player/img/carousel-circuclar-entry-point.png?raw=true"></td>
+    <td><img width="190" height="325" src="https://github.com/ampproject/amphtml/blob/master/src/amp-story-player/img/carousel-circuclar-entry-point.png?raw=true"></td>
```

Comment:
Does this fix the problem? I tried it and they still look squished 😬

<img width="769" alt="Screen Shot 2020-12-03 at 3 58 48 PM" src="https://user-images.githubusercontent.com/3860311/101087619-7500ba00-3580-11eb-8623-f578aa7451b8.png">

Reply:
Hmm, how were you looking at them? Are you using the "rich diff" preview thingy from Github?

Reply:
Can you check again @processprocess ? I think setting fixed sizes was messing it up in some screen sizes.

Reply:
nit: I wonder if this could span the whole width:
<img width="503" alt="Screen Shot 2020-12-04 at 4 23 54 PM" src="https://user-images.githubusercontent.com/3860311/101216234-25d48b00-364d-11eb-87b0-4c71057520f3.png">